### DISCUSSION TITLE
connect-pnponline long connection accessing github cdn?

### DISCUSSION THREAD

**Discussion by har3005** [pnp/powershell#1275] [Q&A]
I've been using pnp.powershell module to access our sharepoint online, using certificate authentication with thumprint and have a very consistent and slow connecion establishing 1min 40 seconds every time from our company network. 
While monitoring the tcp connection, i can see that the powershell process the script runs in attempts to connect to 
some github hosted information the procmon log looks like this for over 50 seconds:
server.xxx.xx:49862 -> cdn-185-199-109-133.github.com:https
what is this about? Can it be disabled or do i have to organise an exception on our firewall to allow this traffic through to get decent connection performance?
Thanks for any input!

**Comment by jefmeyer** [comment]
@har3005 - we're facing the same issues.  Any update on this issue?

**Reply by har3005** [reply]
Good morning,  
indeed, i forgot i posted on GitHub about this! In fact the issue was on our network, SSL inspection Firewall was on for all traffic, that broke the trust to azure with the connect-pnponline command. We excluded SSL inspection to the github.com HTTPS pull from server wlan and all was fine.
@jefmeyer
